### PR TITLE
Add Python tox codebuild module

### PR DESCRIPTION
**Pull request by gds-ahine**

### CODE REVIEW

File: codebuild/codebuild_python_tox/code_build_python_tox.tf [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    privileged_mode             = false
+
+    environment_variable {
+      name  = "AWS_ACCOUNT_ID"
+      value = var.deployment_account_id
+    }
+
+    environment_variable {
+      name  = "PYTHON_SOURCE_DIRECTORY"
+      value = ""
```
Comment (alice-carr) [inline codebuild/codebuild_python_tox/code_build_python_tox.tf:30]: how come you're passing this through as an empty string?

File: codebuild/codebuild_python_tox/code_build_python_tox.tf [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    }
+
+    environment_variable {
+      name  = "PYTHON_VERSION"
+      value = ""
+    }
```
Comment (alice-carr) [inline codebuild/codebuild_python_tox/code_build_python_tox.tf:36]: same question as above

File: codebuild/codebuild_python_tox/code_build_python_tox.tf (hunk 1 above)
Comment (gds-ahine) [reply codebuild/codebuild_python_tox/code_build_python_tox.tf:30]: ahh I wasn't sure about this, I figured you would only know the Python version and the source code directory when you're using the module e.g.

```
module "keycloak_codepipeline_module" {
  source                  = "../../modules/codepipeline"
  environment             = "test"
  account_id              = "103495720024"
  python_source_directory = "3.7.5"
  python_version          = "config_builder"
}
```

**Review by alice-carr — COMMENTED** [review]
It looks all good, I just have a couple of questions about the env vars

File: codebuild/codebuild_python_tox/code_build_python_tox.tf (hunk 1 above)
Comment (alice-carr) [reply codebuild/codebuild_python_tox/code_build_python_tox.tf:30]: ah I see - in that case should we pass them through as variables, ie `value = var.python_source_directory`

Comment (gds-ahine) [reply codebuild/codebuild_python_tox/code_build_python_tox.tf:30]: yep that makes more sense, thanks will get that updated.

**Review by alice-carr — APPROVED** [review]
lgtm

**Pull request merged**